### ISSUE TITLE
Warning in torch.lr_scheduler

### ISSUE THREAD

**Issue by CrafterKolyan** [catalyst-team/catalyst#989]
## 🐛 Bug Report
On catalyst training with scheduler a warning shows.

```
OS: Windows
Python 3.7.8
catalyst==20.10.1
torch==1.6.0
```

### How To Reproduce
```
import torch
from torch import nn
import torch.nn.functional as F
from torch.utils.data import TensorDataset, DataLoader
from catalyst import dl

X = torch.randn(1000, 10)
y = torch.rand(X.shape[0])
model = nn.Linear(X.shape[1], 1)
optimizer = torch.optim.Adam(model.parameters())
scheduler = torch.optim.lr_scheduler.ExponentialLR(optimizer, gamma=0.9)

dataset = TensorDataset(X, y)

loaders = {
    'train': DataLoader(dataset, batch_size=32, shuffle=True),
    'valid': DataLoader(dataset, batch_size=32)
}

class CustomRunner(dl.SupervisedRunner):
    def _handle_batch(self, batch):
        # model train/valid step
        y_pred = self.model(batch['features']).view(-1)

        loss = F.mse_loss(y_pred, batch['targets'])
        self.batch_metrics.update(
            {"loss": loss}
        )

runner = CustomRunner()

runner.train(
    model=model,
    loaders=loaders,
    optimizer=optimizer,
    scheduler=scheduler,
    num_epochs=20
)
```

#### Screenshots
![image](https://user-images.githubusercontent.com/9883873/98355950-eb6ad480-2033-11eb-9396-b6a81f9dada9.png)

### Expected behavior
No warnings

PS
- [x] I know, that I could [join Catalyst slack (#__questions channel)](https://join.slack.com/t/catalyst-team-core/shared_invite/zt-d9miirnn-z86oKDzFMKlMG4fgFdZafw) for issue discussion.

**Comment by and-kul** [comment]
https://github.com/catalyst-team/catalyst/pull/980 has chances to fix this problem

**Comment by CrafterKolyan** [comment]
Should be fixed now
